frontend/app/routes/$lang/_protected/access-to-governmental-benefits/view.tsx

```diff
@@ -88,6 +88,12 @@ export default function AccessToGovernmentalsBenefitsView() {
   const hasDentalPlans = personalInformation.federalDentalPlanId ?? personalInformation.provincialTerritorialDentalPlanId;
 
   if (hasDentalPlans) {
+    const federalSocialProgramName =
+      federalSocialProgramsList.filter((federalSocialProgram) => federalSocialProgram.id === personalInformation.provincialTerritorialDentalPlanId).map((federalSocialProgram) => getNameByLanguage(i18n.language, federalSocialProgram))[0] ?? ' ';
+    const provincialAndTerritorialProgramName =
+      provincialAndTerritorialProgramsList
+        .filter((provincialAndTerritorialProgram) => provincialAndTerritorialProgram.id === personalInformation.provincialTerritorialDentalPlanId)
```

Comment:
We can do this for this PR or a subsequent, but I suggest that the `loader(..)` perform the filtering/mapping of `federalSocialProgramsList`/`provincialAndTerritorialProgramsList` instead of the component.

The `loader(..)` should only return the user's `federalSocialProgram` and `provincialAndTerritorialProgram` - no need to return the two lists to the client-side.

Reply:
That makes sense. Implemented in commit ca88cab054651d5f009e974a192602f99e335f75, since it leaves less in those two branches that I'm going to refactor.